tests/CMakeLists.txt

```diff
@@ -56,12 +56,12 @@ if(AVIF_ENABLE_GTEST)
         find_package(GTest REQUIRED)
     endif()
 
-    add_executable(avifgridapitest avifgridapitest.cc)
+    add_executable(avifgridapitest avifgridapitest.cc aviftest_helpers.cc)
     target_link_libraries(avifgridapitest avif ${AVIF_PLATFORM_LIBRARIES} ${GTEST_BOTH_LIBRARIES})
     target_include_directories(avifgridapitest PRIVATE ${GTEST_INCLUDE_DIRS})
     add_test(NAME avifgridapitest COMMAND avifgridapitest)
 
-    add_executable(avify4mtest avify4mtest.cc)
+    add_executable(avify4mtest avify4mtest.cc aviftest_helpers.cc)
```

Comment:
for a single file it might be overkill, but this could go into a convenience object lib